### PR TITLE
Feat/pwa support

### LINKED ISSUE #46: Progressive Web App friendly KodaDot
Definition of Progressive Web App https://web.dev/progressive-web-apps/

The goal is to have application installable on Android phones. 
Working offline on account and address book would be optional.

**Create PR to the develop branch.**

- [ ] service worker for offline.
- [x] manifest.json - [changed to site.webmanifest](https://developer.mozilla.org/en-US/docs/Web/Manifest) 
- [x] custom offline page
- [ ] wcag - fully accessible

### PR DESCRIPTION
**Pull request by iamonuwa**
Fixes #46 

https://dashboard-lake.now.sh

### CODE REVIEW

File: dashboard/src/App.vue [hunk 1]
```diff
@@ -1,12 +1,18 @@
...
-  <div id="dashboard">
-    <SidebarMenu class="should-be-sidebar" @toggle="toggleSidebar" />
-    <router-view id="routerview" :class="{'sidebar__active': !sidebarClosed }" />
+  <div>
+    <div id="dashboard" v-if="online">
+      <SidebarMenu class="should-be-sidebar" @toggle="toggleSidebar" />
+      <router-view id="routerview" :class="{'sidebar__active': !sidebarClosed }" />
+    </div>
+    <div v-else>
+      <Offline />
```
Comment (yangwao) [inline dashboard/src/App.vue:8]: I think we want to keep sidebar, a user could be confused what or where page is at, just saying it's offline

**Review by yangwao — CHANGES_REQUESTED** [review]
I think we want to keep sidebar, a user could be confused what or where page is at, just saying it's offline

**Comment by vikiival** [comment]
offline

Hello, I experience trouble with connecting.

Using newest Chrome.

**Comment by iamonuwa** [comment]
Thanks for the review. Fixing this immediately

**Comment by iamonuwa** [comment]
> offline
> 
> Hello, I experience trouble with connecting.
> 
> Using newest Chrome.

I'm unable to reproduce this error on incognito chrome browser

File: dashboard/src/App.vue (hunk 1 above)
Comment (iamonuwa) [reply dashboard/src/App.vue:8]: Make sense. I'll update this now.

**Comment by iamonuwa** [comment]
@yangwao made the requested change.

**Comment by yangwao** [comment]
Can you please make PR against develop? 
I'm happy to merge it, thank you.

**Comment by iamonuwa** [comment]
@yangwao done. I can work on more issues for you if you don't mind. I'll get in PRs faster now more than this. 

This is a cool feature https://github.com/vue-polkadot/apps/issues/71. I did one for burner wallet https://github.com/austintgriffith/burner-wallet-electron.
 And many more. You can DM me on twitter @iamonuwa if you want

**Pull request merged**